docs/getting-started/install-configure.md

```diff
@@ -201,29 +201,110 @@ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.
 </div>
 </div>
 
-## Select Provider
+## Select a Getting Started Configuration
+
+Crossplane goes beyond simply modelling infrastructure primitives as custom
+resources - it enables you to define new custom resources with schemas of your
+choosing. We call these "composite resources" (XRs). Composite resources compose
+managed resources -- Kubernetes custom resources that offer a high fidelity
+representation of an infrastructure primitive, like an SQL instance or a
+firewall rule.
+
+We use two special Crossplane resources to define and configure these new custom
+resources:
+
+- A `CompositeResourceDefinition` (XRD) _defines_ a new kind of composite
+  resource, including its schema. An XRD may optionally _offer_ a claim (XRC).
+- A `Composition` specifies which resources a composite resource will be
+  composed of, and how they should be configured. You can create multiple
+  `Composition` options for each composite resource.
+
+XRDs and Compositions may be packaged and installed as a _configuration_. A
+configuration is a [package] of composition configuration that can easily be
+installed to Crossplane by creating a declarative `Configuration` resource, or
+by using `kubectl crossplane install configuration`. In the examples below we
+will install a configuration that defines a new `CompositePostgreSQLInstance` XR
+that takes a single `storageGB` parameter, and creates a connection `Secret`
+with keys for `username`, `password`, and `endpoint`. A `Configuration` exists
+for each provider that can satisfy a `PostgreSQLInstance`. Let's get started!
 
-Now from your terminal, install and configure a provider for Crossplane to use
-for infrastructure provisioning:
 <ul class="nav nav-tabs">
-<li class="active"><a href="#provider-tab-aws" data-toggle="tab">AWS</a></li>
-<li><a href="#provider-tab-gcp" data-toggle="tab">GCP</a></li>
-<li><a href="#provider-tab-azure" data-toggle="tab">Azure</a></li>
-<li><a href="#provider-tab-alibaba" data-toggle="tab">Alibaba</a></li>
+<li class="active"><a href="#aws-tab-1" data-toggle="tab">AWS (Default VPC)</a></li>
+<li><a href="#aws-new-tab-1" data-toggle="tab">AWS (New VPC)</a></li>
+<li><a href="#gcp-tab-1" data-toggle="tab">GCP</a></li>
+<li><a href="#azure-tab-1" data-toggle="tab">Azure</a></li>
 </ul>
 <br>
 <div class="tab-content">
-<div class="tab-pane fade in active" id="provider-tab-aws" markdown="1">
+<div class="tab-pane fade in active" id="aws-tab-1" markdown="1">
 
-### Install AWS Provider
+### Install Configuration Package
 
+> If you prefer to see the contents of this configuration package and how it is
+> constructed prior to install, skip ahead to the [create a configuration]
+> section.
```

Comment:
Added this note to each of the configuration package install commands so folks to jump ahead to see how these exact packages were constructed.